Update site-list-permissions.md (#9594)
microsoftgraph/microsoft-graph-docs-contrib: api-reference/v1.0/api/site-list-permissions.md

Get the [permission](../resources/permission.md) resources from the permissions navigation property on a site.

> [!IMPORTANT]

> > Using GET to list permissions of SharePoint subsites is not supported at this time.

Reviewer: ```suggestion
> Using the LIST method to retrieve SharePoint subsites' permissions isn't supported at this time.
```
